### ISSUE TITLE
Hard crash writing out Component Loads Summary table. 

### ISSUE THREAD

**Issue by EnergyArchmage** [NREL/EnergyPlus#5398]
Defect file crashes at line 12147 in OutputReportTabular.cc while writing out the component loads table.  

```
  totalColumn( rDOAS ) += CalcZoneSizing( CoolDesSelected, iZone ).DOASHeatAddSeq( timeHeatMax );
```

Test file from interface developer has cooling termostat setpoint set very high which seems to cause CoolDesSelected to be zero, leading to crash.   Component Loads summary needs to finish gracefully when design loads are zero and no design day was selected.

**Comment by mjwitte** [comment]
This may be a duplicate of #5357 .  Looks like the fix is here #5388 Or this may be a different case.

**Comment by Myoldmopar** [comment]
I'll close this as a duplicate of #5357, but either way, I am merging in #5388.
